plugins/ftdidmx/FtdiDmxDevice.cpp

```diff
@@ -56,15 +56,19 @@ FtdiDmxDevice::~FtdiDmxDevice() {
 bool FtdiDmxDevice::StartHook() {
   unsigned int interface_count = m_widget->GetInterfaceCount();
   unsigned int successfully_added = 0;
+  unsigned int serial = 0;
 
   OLA_INFO << "Widget " << m_widget->Name() << " has " << interface_count
            << " interfaces.";
 
   for (unsigned int i = 1; i <= interface_count; i++) {
+    if (!StringToInt(m_widget->Serial(), &serial, false, 36)) {
```

Comment:
You're allowing trailing characters with strict=false, but if you buy two dongles at the same time, the leading ones are more likely to be the same, so truncating before passing to StringToInt would be more likely to generate unique UIDs.

Reply:
You are right, I should just take the last 6 characters.

I probably also should now disallow dongles without a serial?

Reply:
Did a change let me know what you think.